code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/api/action/AddServiceLinkActionHandler.java

```diff
@@ -52,7 +52,7 @@ protected void createMap(long serviceId, long consumedServiceId) {
                     SERVICE_CONSUME_MAP.SERVICE_ID,
                     serviceId, SERVICE_CONSUME_MAP.CONSUMED_SERVICE_ID, consumedServiceId);
         }
-        objectProcessManager.executeProcess(ServiceDiscoveryConstants.PROCESS_SERVICE_CONSUME_MAP_CREATE,
+        objectProcessManager.scheduleProcessInstance(ServiceDiscoveryConstants.PROCESS_SERVICE_CONSUME_MAP_CREATE,
```

Comment:
@cjellick  Today handler just creates a map record in the DB; but once actual DNS configuration is in, there will also be a post handler for map.create that actually programs the network agent. I didn't want the handler to wait for the result